src/components/BadgeBuilderDialog.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+    <form @submit.prevent="submit">
+        <div ref="modal" class="modal fade" tabindex="-1" data-bs-backdrop="static">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <h5 id="exampleModalLabel" class="modal-title">
+                            {{ $t("Generate Badge") }}
+                        </h5>
+                        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close" />
+                    </div>
+                    <div class="modal-body">
```

Comment:
Please add an [info alert](https://getbootstrap.com/docs/5.3/components/alerts/) here to inform users that

> Note that adding badges at a monitor level is [possible too if they are on any public status page](https://github.com/louislam/uptime-kuma/wiki/Badge)